File: pgm/ABC_Visual_Basic/ABC_152_B.vb
Public Class Main
    Public Shared Sub Main(ByVal args() As String)
        Dim a, b As Char
        Dim arr(2) As String
        arr = Split(Console.ReadLine())
        a = arr(0) : b = arr(1)
        If a < b Then
            Console.WriteLine(New String(a, Integer.Parse(b)))
        Else
            Console.WriteLine(New String(b, Integer.Parse(a)))
        End If
    End Sub
End Class

File: pgm/ABC_C#/ABC_152_B.cs
using System;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int a = int.Parse(arr[0]), b = int.Parse(arr[1]);
        Console.WriteLine(
            a < b ? new string(arr[0][0], b) : new string(arr[1][0], a));
    }
}